### ISSUE TITLE
Tabbing out from rangeSelector date input does not apply the date

### ISSUE THREAD

**Issue by kamil-musialowski** [highcharts/highcharts#21130]
#### Expected behaviour
Adding the date via keyboard and tabbing out from the input should apply the date in rangeSelector input date.

#### Actual behaviour
Tabbing out brings back the old value

#### Live demo with steps to reproduce
https://jsfiddle.net/BlackLabel/u97g8mr4/

1. Mouse click on the "From" input
2. Enter some day with the keyboard
3. Click TAB to leave the input

Value went back to the old one

#### Product version
Stock 11.4.1
